source/Cosmos.Core/CPU.cs

```diff
@@ -37,6 +37,16 @@ public static uint GetAmountOfRAM()
         [PlugMethod(PlugRequired = true)]
         public static uint GetEBPValue() => throw null;
 
+        /// <summary>
+        /// Get position of current ESP register
+        /// </summary>
+        /// <returns></returns>
+        [PlugMethod(PlugRequired = true)]
```

Comment:
i feel this should not be exposed to c#

Reply:
yes u right it was a bug test

Reply:
The Change is set the methods from public to internel